### ISSUE TITLE
Error on npm run prod

### ISSUE THREAD

**Issue by WhishpersOfThePast** [CupOfTea696/laravel-mix-imagemin#3]
For not write a lot of stuff in mix file decided to try this package and all was pretty nice before I run `npm run prod` and always obtain error from font file.
`npm run watch` and `npm run dev` works perfectly.
I tryed playing with public path without results.
Without package `npm run prod` dont have any error
mix file
```js
const mix = require('laravel-mix');
require('laravel-mix-imagemin');

mix
  .js('resources/js/app.js', 'public/js')
  .extract([
    'jquery',
    'bootstrap/js/dist/collapse',
    'bootstrap/js/dist/dropdown',
    'bootstrap/js/dist/carousel',
  ])
  .sass('resources/sass/app.scss', 'public/css')
  .imagemin(
    'images/**/**.*', {
      context: 'resources',
    }, {
      optipng: {
        optimizationLevel: 4
      },
      jpegtran: null,
      plugins: [
        require('imagemin-mozjpeg')({
          quality: 80,
          progressive: true,
        }),
      ],
    }
  );

if (!mix.inProduction()) {
  mix.sourceMaps();
}

if (mix.inProduction()) {
  mix.version();
}
```

> @ production /home/...
cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --no-progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js
 DONE  Compiled successfully in 82145ms                                                                                                                                       17:18:58
Error: ENOENT: no such file or directory, open '/home/.../public/fonts/GothamBold.woff2?fe075cb2c808644d75839c8d42fc40ea'
    at Object.fs.openSync (fs.js:646:18)
    at Object.fs.readFileSync (fs.js:551:33)
    at File.read (/home/.../node_modules/laravel-mix/src/File.js:166:19)
    at File.version (/home/.../node_modules/laravel-mix/src/File.js:173:25)
    at Manifest.hash (/home/.../node_modules/laravel-mix/src/Manifest.js:52:65)
    at manifest.forEach.file (/home/.../node_modules/laravel-mix/src/webpackPlugins/CustomTasksPlugin.js:81:47)
    at Array.forEach (<anonymous>)
    at CustomTasksPlugin.applyVersioning (/home/.../node_modules/laravel-mix/src/webpackPlugins/CustomTasksPlugin.js:81:18)
    at compiler.plugin.stats (/home/.../node_modules/laravel-mix/src/webpackPlugins/CustomTasksPlugin.js:14:22)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/.../node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:18:1)
    at AsyncSeriesHook.lazyCompileHook (/home/.../node_modules/tapable/lib/Hook.js:154:20)
    at emitRecords.err (/home/.../node_modules/webpack/lib/Compiler.js:267:22)
    at Compiler.emitRecords (/home/.../node_modules/webpack/lib/Compiler.js:449:39)
    at emitAssets.err (/home/.../node_modules/webpack/lib/Compiler.js:261:10)
    at hooks.afterEmit.callAsync.err (/home/.../node_modules/webpack/lib/Compiler.js:435:14)
    at _next1 (eval at create (/home/.../node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:16:1)
    at _err1 (eval at create (/home/.../node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:25:1)
    at onAfterEmit (/home/.../node_modules/laravel-mix-imagemin/src/ManifestPlugin.js:12:13)
    at _next0 (eval at create (/home/.../node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:21:1)
    at _err0 (eval at create (/home/.../node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:34:1)
    at compiler.hooks.afterEmit.tapAsync (/home/.../node_modules/copy-webpack-plugin/dist/index.js:116:7)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/.../node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:30:1)
    at AsyncSeriesHook.lazyCompileHook (/home/.../node_modules/tapable/lib/Hook.js:154:20)
    at asyncLib.forEachLimit.err (/home/.../node_modules/webpack/lib/Compiler.js:432:27)
    at /home/.../node_modules/neo-async/async.js:2818:7
    at done (/home/.../node_modules/neo-async/async.js:3522:9)
    at /home/.../node_modules/graceful-fs/graceful-fs.js:45:10
    at FSReqWrap.oncomplete (fs.js:135:15)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @ production: `cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --no-progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @ production script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/gabriel/.npm/_logs/2019-06-07T15_18_58_365Z-debug.log

**Comment by CupOfTea696** [comment]
Where are you referencing that font?

**Comment by WhishpersOfThePast** [comment]
in resources/sass/_fonts.scss
``
@font-face {
  font-family: 'Gotham';
  src        : url("../fonts/GothamMedium.woff2") format('woff2'), url("../fonts/GothamMedium.woff") format('woff');
  font-weight: normal;
  font-style : normal;
}
@font-face {
  font-family: 'GothamBold';
  src        : url("../fonts/GothamBold.woff2") format('woff2'), url("../fonts/GothamBold.woff") format('woff');
  font-weight: bold;
  font-style : normal;
}
@font-face {
  font-family: 'Gotham Book';
  src        : url("../fonts/GothamBook.woff2") format('woff2'), url("../fonts/GothamBook.woff") format('woff');
  font-weight: normal;
  font-style : normal;
}
``
and my font files located in resources/fonts/...

> GothamBold.woff
> GothamBold.woff2
> GothamBook.woff
> GothamBook.woff2
> GothamMedium.woff
> GothamMedium.woff2

**Comment by CupOfTea696** [comment]
This should now be fixed, please update to v1.0.1
